README.md

```diff
@@ -91,18 +91,27 @@ Collect and process real time twitter data to analyse popularity of tweets with
 - nltk
 - networkx
 - streamlit
-- pandas,numpy
+- pandas
+- numpy
 - matplotlib
 
 <br>
 </br>
 
 ## Setup Locally
 
-- Go the the cloned directory on your local machine.
+- Clone the repository on your local machine.
+  ```sh
+  git clone https://github.com/kaustav202/RealTime-TwitterDataAnalysis.git
+  ```
+- Go into the cloned directory
 - Run `pip install -r requirements.txt` to install all the dependencies.
+- Create a developer account on twitter: https://developer.twitter.com/en
 - Get your Twitter API credentials and replace the placeholders in twitter_config.py.
-- run `python main.py` which is the application entry point.
+    - Go to the Twitter Developer Portal Projects & Apps page at https://developer.twitter.com/en/portal/projects-and-apps
+    - Find the API/consumer key and secret under the Consumer Keys section of the Keys and Tokens tab of your app
+    - Your account's access token and secret for your app can be found under the Authentication Tokens section of the Keys and Tokens tab of your app
+- Run `python main.py` which is the application entry point.
```

Comment:
An initial step is required to start the streaming by running stream.py before this. 
`python stream.py`

Reply:
While the streaming continues.. new tweets keep getting added to tweets.json, hence the main file should ideally be run after a gap so that the results are better (more data points).

It is also important to mention that these are two independent steps involved

Reply:
I'll solve this asap